config/config-api/src/main/java/com/thoughtworks/go/domain/config/ConfigurationProperty.java

```diff
@@ -266,14 +266,6 @@ public void initialize() {
         if (configurationValue == null && encryptedValue == null) {
             configurationValue = new ConfigurationValue("");
         }
-
-        if (encryptedValue != null && isNotBlank(encryptedValue.getValue())) {
-            setEncryptedValue(cipher.maybeReEncryptForPostConstructWithoutExceptions(encryptedValue.getValue()));
-        }
-    }
-
-    private void setEncryptedValue(String encryptedValue) {
```

Comment:
Leave this be. Might be used from within rails.

Reply:
Make it a public method, perhaps.

Reply:
Got it back.